compte/views.py

```diff
@@ -149,6 +154,30 @@ def change_email(request, activation_token):
         )
 
 
+@login_required
+def delete_account(request):
+    try:
+        res = remove_personal_informations(request.user, shortuuid.uuid)
+    except RuntimeError as err:
```

Comment:
We must ensure that the request has been submitted in `POST`, ideally validating a form containing a boolean field checked expressing user consent before actually soft-deleting the user account.